### DISCUSSION TITLE
question focntionnement variables d’environnement

### DISCUSSION THREAD

**Discussion by gch-meoss** [mviewer/mviewer#910] [Q&A]
Bonjour,

j'ai testé l'utilisation les fichiers .json pour configurer les variables d’environnement.

dans les balises "<layer.." cela semble bien fonctionner.

par contre dans la balise **"<application..**" cela ne semble pas fonctionner pour les attributs **help** et **style**

est que cela est normal ? avez vous le même comportement ou bien le pb viens de chez moi ? 

merci !

**Comment by Gaetanbrl** [comment]
Bonjour,
Les variables d'environnement ne concernent actuellement que les couches et customlayer :

https://mviewerdoc.readthedocs.io/fr/stable/doc_tech/config_environmentvar.html

Les autres URLs ne sont pas modifiée. C'est toutefois faisable mais cela demande d'apporter une évolution dans mviewer :

https://github.com/mviewer/mviewer/blob/master/js/configuration.js#L620

**Comment by gch-meoss** [comment]
merci pour le retour rapide !!

je vais regarder le fichier configuration.js mais ma maîtrise de js n'est pas optimale ^^"

**Comment by Gaetanbrl** [comment]
Pouvez-vous faire une issue qui contient votre demande d'évolution ? 
Merci :)

**Reply by gch-meoss** [reply]
ok je m'en occupe, merci

**Reply by gch-meoss** [reply]
en fouillant un peu plus il ya déjà une issue concernant ce sujet. j'y rajoute un commentaire  :)
(https://github.com/mviewer/mviewer/issues/728)
